fastapi/__init__.py

```diff
@@ -1,7 +1,6 @@
 """FastAPI framework, high performance, easy to learn, fast to code, ready for production"""
 
 __version__ = "0.61.0"
-
```

Comment:
Readd this line, please

Reply:
This diff was generated by running `bash scripts/format-imports.sh`. I just confirmed by re-adding it and running the format script; it removes the line again.